### ISSUE TITLE
Won't start with error 'ImportError: cannot import name GConf'

### ISSUE THREAD

**Issue by gabrielsr** [projecthamster/hamster#251]
** (hamster:28026): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-EuA24KyooW: Connection refused
Traceback (most recent call last):
  File "/usr/bin/hamster", line 30, in <module>
    from hamster import client, reports
  File "/usr/lib/python2.7/dist-packages/hamster/reports.py", line 32, in <module>
    from hamster.lib.configuration import runtime
  File "/usr/lib/python2.7/dist-packages/hamster/lib/configuration.py", line 33, in <module>
    from gi.repository import GConf as gconf
ImportError: cannot import name GConf

**Comment by gabrielsr** [comment]
sudo apt-get install gir1.2-gconf-2.0 gir1.2-glib-2.0

solved the problem.

**Comment by toupeira** [comment]
Yep sorry this was missing in the README, see https://github.com/projecthamster/hamster/pull/212 :)
